### ISSUE TITLE
Implement backup strategy for Kafka

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-base/RADAR-Docker#171]
We could implement a strategy for backing up Kafka information like explained here: https://jobs.zalando.com/tech/blog/backing-up-kafka-zookeeper/.
They use S3 connector for Kafka topic contents and Burry for Zookeeper snapshots.

**Comment by yatharthranjan** [comment]
Yes this was also mentioned in #106. It will definitely be great to have
